lms/templates/main.html

```diff
@@ -137,7 +137,7 @@
     % endif
 
     <div class="content-wrapper" id="content">
-      ${self.body()}
+      ${next.body()}
```

Comment:
This looks good to me. It allows for having multiple templates in the inheritance chain.